doc/guides/west/build-flash-debug.rst

```diff
@@ -79,10 +79,11 @@ Finally, you can add additional arguments to the CMake invocation performed by
 command. For example, to use the Unix Makefiles CMake generator instead of
 Ninja (which ``west build`` uses by default), run::
 
-  west build -- -G'Unix Makefiles'
+  west build -b reel_board samples/hello_world -- -G'Unix Makefiles'
```

Comment:
Previously you said the source directory path is the last option, but in this example it isn't (there are more options after the source directory path (``-- -G'Unix Makefiles'``)

Reply:
I've rephrased that to use positional arguments instead. `west` invokes `cmake` and everything that comes after `--`, `west` passes on to `cmake` directly.